yarn.lock

```diff
@@ -1081,63 +1081,63 @@
     "@babel/helper-validator-identifier" "^7.16.7"
     to-fast-properties "^2.0.0"
 
-"@backstage/app-defaults@^1.0.1":
```

Comment:
These changes probably don't belong in this PR :)

Reply:
These are just the latest versions pulled in when running yarn install but I reverted the change.